src/main/java/org/junitpioneer/internal/PioneerAnnotationUtils.java

```diff
@@ -130,11 +132,12 @@ public static boolean isAnyRepeatableAnnotationPresent(ExtensionContext context,
 	 */
 	public static <A extends Annotation> List<Annotation> findAnnotatedAnnotations(AnnotatedElement element,
 			Class<A> annotation) {
+		boolean isRepeated = annotation.isAnnotationPresent(Repeatable.class);
```

Comment:
Checking if an annotation is repeatable doesn't mean that it is used multiple times, so I think the variable must be named `isRepeatable`

Reply:
Makes sense, will do